itl/command-plugins.conf

```diff
@@ -900,6 +900,10 @@ object CheckCommand "imap" {
 			value = "$imap_timeout$"
 			description = "Seconds before connection times out (default: 10)"
 		}
+		"--sni" = {
+			value = "$ssl_sni$"
```

Comment:
Are you sure about this?

Reply:
Oh sorry, my bad! Fixed in 92f4c4c39. However, I've not changed the description since it has a comparable description in the [monitoring-plugins documentation](https://www.monitoring-plugins.org/doc/man/check_imap.html): `SSL server_name`